server/handle_map_layer_zxy.go

```diff
@@ -156,8 +156,36 @@ func (req HandleMapLayerZXY) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		m = m.AddDebugLayers()
 	}
 
-	encodeCtx := context.WithValue(r.Context(), observability.ObserveVarMapName, m.Name)
-	pbyte, err := m.Encode(encodeCtx, tile)
+	ctx := context.WithValue(r.Context(), observability.ObserveVarMapName, m.Name)
+	var params map[string]string
+
+	// check for query parameters and populate param map with their values
+	if req.Atlas.HasParams(req.mapName) {
+		params = make(map[string]string)
+		err = r.ParseForm()
```

Comment:
Any reason why you went with r.ParseForm() instead of using [r.URL.Query()](https://pkg.go.dev/net/url#URL.Query)?